starknet_py/net/schemas/common.py

```diff
@@ -336,3 +336,17 @@ class StorageEntrySchema(Schema):
     def make_dataclass(self, data, **kwargs):
         # pylint: disable=no-self-use
         return StorageEntry(**data)
+
+
+class ChainIdField(fields.Field):
+    def _deserialize(self, value, attr, data, **kwargs):
+        if isinstance(value, int):
```

Comment:
Update: after online discussion we decided to accept `chainId` only as `str`. It could be extended to accept `int` in future, if there is such a need.